auto_nag/scripts/not_landed.py

```diff
@@ -32,13 +36,31 @@ def description(self):
     def has_assignee(self):
         return True
 
+    def get_autofix_change(self):
+        return self.changes_per_bug
+
     def get_extra_for_template(self):
         return {"nweeks": self.nweeks}
 
     def get_extra_for_needinfo_template(self):
         self.extra_ni.update(self.get_extra_for_template())
         return self.extra_ni
 
+    def make_comment(self, bugid, nick):
+        template_name = self.needinfo_template()
+        assert bool(template_name)
+        env = Environment(loader=FileSystemLoader("templates"))
+        template = env.get_template(template_name)
+
+        comment = template.render(
+            nickname=nick,
```

Comment:
The comment will only mention the author, but there will be a needinfo to both the author and a reviewer. Could it be confusing? Let's try and see if somebody complains.

Reply:
Agreed, I don't know what could be a good sentence.